### PR TITLE
DEF-235: Add allowed tiers input for Pico/Coral SSO to Irving Integrations settings page

### PR DESCRIPTION
**Pull request by tylerreckart**
Adds a new input where users can add comma-separated values to pass whitelisted tiers to the `<PicoObserver />` component on the front-end.

### CODE REVIEW

File: inc/integrations/class-pico.php [hunk 1]
```diff
@@ -77,6 +78,15 @@ public function register_settings_fields() {
 			'wp_irving_integrations',
 			'irving_integrations_settings'
 		);
+
+		// Register new fields for the Coral integration.
+		add_settings_field(
+			'wp_irving_pico_tiers',
+			esc_html__( 'Pico/Coral Whitelisted SSO Tiers', 'wp-irving' ),
```
Comment (joemcgill) [inline inc/integrations/class-pico.php:85]: Avoid "Whitelist" in favor of more inclusive language, here and throughout 🔢 

```suggestion
			esc_html__( 'Allowed Pico/Coral SSO Tiers', 'wp-irving' ),
```

File: inc/integrations/class-pico.php [hunk 2]
```diff
@@ -91,6 +101,19 @@ public function render_pico_auth_key_input() {
...
 
+	/**
+	 * Render an input for the Pico Whitelisted SSO Tiers.
+	 */
+	public function render_pico_tiers_input() {
+		// Check to see if there are existing tiers in the option.
+		$tiers = $this->options[ $this->option_key ]['tiers'] ?? '';
+
+		?>
+			<input type="text" name="irving_integrations[<?php echo esc_attr( 'pico_tiers' ); ?>]" value="<?php echo esc_attr( $tiers ); ?>" />
```
Comment (joemcgill) [inline inc/integrations/class-pico.php:112]: We may want to add some help text that explains that this needs to be a comma separated list.

**Review by joemcgill — APPROVED** [review]
Couple of comments, but pre-approving pending updates. 🌵

File: inc/integrations/class-pico.php (hunk 1 above)
Comment (tylerreckart) [reply inc/integrations/class-pico.php:85]: Great point! I hadn't even considered the terms lack of inclusivity.

**Pull request merged**